<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_3</class>
 <widget class="QDialog" name="fct_comp_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liste des épreuves d'une catégorie (version 2, catégories extraites de la BD)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Saisissez une catégorie :</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QComboBox" name="comboBox_fct_3_categorie">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>650</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Travail à réaliser : améliorer l'interface afin de proposer une liste des catégories possibles (issues de la base de données).</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="table_fct_comp_3">
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>numEp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nomEp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>formeEp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nomDi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nbSportifsEp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dateEp</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_fct_comp_3">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_fct_3_categorie</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_3</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>-133</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
 </slots>
</ui>
